js/scripts/build-rpc-markdown.js

```diff
@@ -292,7 +285,19 @@ Object.keys(interfaces).sort().forEach((group) => {
   const tocMain = [];
   const tocSections = {};
 
-  Object.keys(spec).sort().forEach((iname) => {
+  // Comparator that will sort by sections first, names second
+  function methodComparator (a, b) {
+    const sectionA = spec[a].section || '';
+    const sectionB = spec[b].section || '';
+
+    if (sectionA === sectionB) {
```

Comment:
This will produce unexpected results if `sectionA.localeCompare(sectionB) === 0`.